tests/tasks/test_example.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from openedx_webhooks.tasks.example import add
+
+
+def test_add():
+    assert add(5, 6) == 11
+
+
+if __name__ == '__main__':
```

Comment:
My instinct was the same, but again, it wasn't clear to me what the edX standards are. I thought I would err on the side of more formal documentation.